lib/skc_ast2hir/src/ctx_stack.rs

```diff
@@ -342,6 +352,15 @@ impl<'hir_maker> Iterator for LVarIter<'hir_maker> {
                     is_lambda_scope: false,
                 })
             }
+            HirMakerContext::If(if_ctx) => {
+                self.finished = true;
```

Comment:
`finished` is set to true when there is no more outer ctx to find local variable.
This should be `self.cur -= 1` because we want to proceed to the next ctx surrounding the `if`.

Reply:
Aahh ok, I will fix that. Thank you.